### DISCUSSION TITLE
第2节课作业(24班)

### DISCUSSION THREAD

**Discussion by zhjunqin** [InternLM/Tutorial#270] [作业]
第2节课作业(24班)

**Comment by yezhou05** [comment]
https://app.yinxiang.com/fx/50b4858d-0caf-41a8-b0b8-fb426fd491e9

**Comment by kelenlv** [comment]
https://github.com/kelenlv/LLM_notes/blob/main/Day2.md

**Comment by waurblove** [comment]
https://github.com/waurblove/LLM/blob/main/day2.md

**Comment by ljn12yyds** [comment]
https://blog.csdn.net/ljn20001229/article/details/135510235?spm=1001.2014.3001.5502

**Comment by lindsey-chang** [comment]
https://blog.csdn.net/qq_37397652/article/details/135540633?spm=1001.2014.3001.5501

**Comment by yansuoyuli** [comment]
https://github.com/yansuoyuli/LLM-chapter2/blob/main/readme.md

**Comment by ch521-od** [comment]
https://blog.csdn.net/qq_74261455/article/details/135573377?spm=1001.2014.3001.5502

**Comment by PopoData** [comment]
https://zhuanlan.zhihu.com/p/677677719

**Comment by aaashinshin** [comment]
https://blog.csdn.net/weixin_44254889/article/details/135575917

**Comment by williamleo0730** [comment]
https://zhuanlan.zhihu.com/p/677666175

**Comment by robotzm** [comment]
https://blog.csdn.net/weixin_48319544/article/details/135576497?spm=1001.2014.3001.5502

**Comment by flh1107** [comment]
第2课作业：https://www.bilibili.com/read/cv29532674/

**Comment by juju0706** [comment]
第2节课作业：https://zhuanlan.zhihu.com/p/678206698

**Comment by Xinyi-Cheng** [comment]
https://zhuanlan.zhihu.com/p/678388737
